docs/source/cli_help/github.rst

```diff
@@ -25,6 +25,16 @@ a GitHub API token with ``repo`` scope is required, see https://github.com/setti
   Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
   Equivalent to ``git machete anno --sync-github-prs``.
 
+``checkout-pr``:
+
+  Check out the head branch of the given pull request (specified by number),
+  also traverse chain of pull requests upwards, adding branches one by one to git-machete and check them out locally.
+  Once pull request is checked out locally, add new branches to the definition file according to the PR structure and annotate them with pull request numbers.
```

Comment:
The same thing mentioned twice: _adding branches one by one to git-machete_ vs  _adding branches one by one to git-machete and check them out locally_, pls condense